ucsb-cs156-f24/proj-happycows-f24-10
# Add Entity, Repository and Migration for Course table

## Overview
In this PR I add an entity file, repository and database migration for the Course database table. The database schema is described in #11.

## Validation
To test this change:
 1. go to the dokku and access database happycows-dev-r0ckwav3-db via `dokku postgres:connect`
 2. verify that the table exists with `\dt`
 3. verify that the columns are correct with `\d courses`

## Tests


## Linked Issues

Closes #30
